tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -113,6 +117,14 @@ def delete_and_create_osd_node(osd_node_name):
 
     old_osd_node_names = node.get_osd_running_nodes()
 
+    # If the cluster is an MS provider cluster, and we also have MS consumer clusters in the run
+    if is_ms_provider_cluster() and config.is_consumer_exist():
+        pytest.skip(
```

Comment:
Why it is not skipped by bugzilla marker so that we don't have to track it manually?